documentation/assemblies/managing/assembly-drain-cleaner.adoc

```diff
@@ -0,0 +1,77 @@
+// This assembly is included in the following assemblies:
+//
+// using/assembly-management-tasks.adoc
+
+[id='assembly-drain-cleaner-{context}']
+= Evicting pods with Strimzi Drain Cleaner
+
+[role="_abstract"]
```

Comment:
I think the abstract content in this assembly can actually be a module. There is a lot of conceptual info, in this I think.

Reply:
Going to leave as is for now.